Utilities/Create-AzureConfigFilesForJenkins.ps1

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+<#
+.SYNOPSIS
+    This file creates 'Property Files' for jenkins dynamic menu creation.
+    These 'Property Files' can be used to create dynamic jenkins menus using parameter type - "Extended Choice Parameter".
+
+.PARAMETER
+	See source code for the detailed parameters
+
+.NOTES
+	PREREQUISITES:
+	1) You should have a valid Azure subscription and read access.
+
+.EXAMPLE
+	Example 1 : Create Jenkins Property files
+	.\Utilities\Create-AzureConfigFilesForJenkins.ps1
+#>
+
+param (
+    [string] $SecretFilePath = "",
+    [string] $ImageLocation = "westus2",
+    [string] $ImagePublishers = "Canonical,SUSE,Oracle,CoreOS,RedHat,OpenLogic,credativ,kali-linux,clear-linux-project,MicrosoftOSTC,MicrosoftRServer,MicrosoftSharePoint,MicrosoftSQLServer,MicrosoftVisualStudio,MicrosoftWindowsServer,MicrosoftWindowsServerEssentials,MicrosoftWindowsServerHPCPack,MicrosoftWindowsServerRemoteDesktop"
+)
+
+#region Authenticate the powershell session.
+if ($SecretFilePath) {
+    .\Utilities\AddAzureRmAccountFromSecretsFile.ps1 -customSecretsFilePath $SecretFilePath
+} else {
+    Write-Host "Secret File not provided. Checking subscriptions..."
+    $Subscriptions = Get-AzSubscription
```

Comment:
What if the system has Az and AzRm both cmdlets, it should show conflict. Please check out that situation.

Reply:
Microsoft recommends to not to have AzureRm and Az modules installed side by side.

> At this point, you might want to run the Uninstall-AzureRM cmdlet provided in the Az module, just to make sure that all versions of AzureRM have been uninstalled and won't cause conflicts.

Reference - https://docs.microsoft.com/en-us/powershell/azure/migrate-from-azurerm-to-az?view=azps-2.3.2

And our LISAv2 is also using Az module. This script is compliant with existing LISAv2.